Documentation for Delete Credential action and related changes (#31719)
keycloak/keycloak: docs/documentation/upgrading/topics/changes/changes-24_0_3.adoc

The full wildcard `*` can still be used as a valid redirect in development for http(s) URIs with those characteristics. In production environments a exact valid redirect URI without wildcard needs to be configured for any URI of that type.

Please note that wildcard valid redirect URIs are not recommended for production and not covered by the OAuth 2.0 specification.

= Deprecated Account REST endpoint for removing credential

> The Account REST endpoint for removing the credential of the user is deprecated. Account console does not use this endpoint from this version. It is replaced by the `Delete Credential` application-initiated
> action, which is triggered by Account console when user want to remove the credential of some user.

Reviewer: ```suggestion
The Account REST endpoint for removing the credential of the user is deprecated. Starting at this version, the Account Console no longer uses this endpoint. It is replaced by the `Delete Credential` application-initiated
action, which is triggered by Account Console when a user want to remove the credential of a user.
```
